.ci/checker-framework.groovy

```diff
@@ -112,27 +112,31 @@ private static int checkCheckerFrameworkReport(final String profile) {
  */
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
+    final List<List<String>> checkerFrameworkErrors = new ArrayList<>()
     final String command = "mvn -e --no-transfer-progress clean compile" +
         " -P${profile},no-validations"
     final ProcessBuilder processBuilder = new ProcessBuilder(getOsSpecificCmd(command).split(' '))
     processBuilder.redirectErrorStream(true)
     final Process process = processBuilder.start()
-
     BufferedReader reader = null
     try {
         reader = new BufferedReader(new InputStreamReader(process.inputStream))
         String lineFromReader = reader.readLine()
         while (lineFromReader != null) {
             println(lineFromReader)
             checkerFrameworkLines.add(lineFromReader)
+            if (lineFromReader.contains("OutOfMemoryError")) {
```

Comment:
We need any error, can we make code general ?
Can we simply rely on `mvn` exit code?

Reply:
Okay.

Reply:
> We need any error, can we make code general ? Can we simply rely on `mvn` exit code?

I am trying to throw an exception the moment it encounters a non-zero exit code, assuming a OOM detected. And, this probably might make the CI/CD to fail. Please let me know your thoughts on it.